manual/manual/cmds/runtime.etex

```diff
@@ -126,14 +126,13 @@ The following environment variables are also consulted:
         This option takes no argument.
   \item[h] The initial size of the major heap (in words).
   \item[a] ("allocation_policy") The policy used for allocating in the
-  OCaml heap.  Possible values are 0 for the next-fit policy, and 1
-  for the first-fit policy.  Next-fit is usually faster, but first-fit
-  is better for avoiding fragmentation and the associated heap
-  compactions.
+    major heap. See the Gc module documentation for details.
```

Comment:
I think that we should at least enumerate the possible values for the option, before pointing to the Gc module documentation.

Reply:
The slightly annoying thing with this choice is that it needs to be updated if we add a new policy again. Given the very small amount of details for most of the other option, I think that it's easier to just refer to the Gc manual (which surely people remember to update when they change the Gc?).

Reply:
Updating this list for every new policies doesn't sound like a gruesome burden? And I think it is better if the reader has at least the information about which values are possible before following the link.

Reply:
Fine, I reused the manpage's description. (I tried to just enumerate the possible values, but it just looked like bad documentation.)

Reply:
Thanks!